integration-cli/docker_cli_volume_test.go

```diff
@@ -25,7 +25,7 @@ func (s *DockerSuite) TestVolumeCliCreateOptionConflict(c *check.C) {
 	c.Assert(err, check.NotNil, check.Commentf("volume create exception name already in use with another driver"))
 	c.Assert(out, checker.Contains, "A volume named test already exists")
 
-	out, _ = dockerCmd(c, "volume", "inspect", "--format='{{ .Driver }}'", "test")
+	out, _ = dockerCmd(c, "volume", "inspect", "--format={{ .Driver }}", "test")
```

Comment:
There was some discussion earlier in this issue about this, starting here: https://github.com/docker/docker/pull/22769#issuecomment-220720692

These single quotes don't actually provide any value here. When run from a shell they inform the shell that the string should be read as a literal string, but from an exec that is not relevant. Adding the quotes doesn't get us any closer to the "real behaviour" of this command (as run by a user).

There is code in mflags (`0e9c40e`) that explicitly strips off these unnecessary quotes. The consensus was that we didn't need to port this into pflag because it's not doing anything. The behaviour is already handled by a shell, and outside of a shell they are unnecessary.